src/main/java/frc/robot/subsystems/arm/ArmStageOne.java

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems.arm;
+
+import com.ctre.phoenix.motorcontrol.FeedbackDevice;
+import com.ctre.phoenix.motorcontrol.InvertType;
+import com.ctre.phoenix.motorcontrol.TalonSRXControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj.DutyCycleEncoder;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+import static frc.robot.Constants.ArmConstants;
+
+public class ArmStageOne extends SubsystemBase {
+  private TalonSRX armMotorPrimary;
+  private TalonSRX armMotorSecondary;
+  private DutyCycleEncoder magEncoder;
```

Comment:
mag encoder will be plugged into talon SRX, I don't think we need to instantiate it as a duty cycle encoder do we?

Reply:
I think you're right. I can replace its getPosition calls with armMotorPrimary.getSelectedSensorPosition(). That should work, right?